src/types/events/index.ts

```diff
@@ -28,8 +28,19 @@ interface EnvelopedEvent<Event = BasicSlackEvent> extends StringIndexed {
   type: 'event_callback';
   event_id: string;
   event_time: number;
-  // TODO: is this optional?
-  authed_users: string[];
+  // TODO: the two properties below are being deprecated on Feb 24, 2021
+  authed_users?: string[];
+  authed_teams?: string[];
```

Comment:
We didn't have authed_teams before but it seems like it should have been here. Is this right @seratch?

Reply:
Once the `authorizations` is enabled, both `authed_users` and `authed_teams` are not available here. That said, there is no reason not to have `authed_teams` while having `authed_users`. Good catch 👍